src/frontend/src/index.ts

```diff
@@ -94,6 +97,28 @@ const init = async () => {
   // Prepare the actor/connection to talk to the canister
   const connection = new Connection(readCanisterId());
 
+  // Figure out if user is trying to add a device. If so, use the anchor from the URL.
+  const addDeviceAnchor = getAddDeviceAnchor();
+  if (addDeviceAnchor !== undefined) {
+    // Register this device (tentatively)
+    await registerTentativeDevice(addDeviceAnchor, connection);
+
+    // Show a good bye message
+    await new Promise((_) => {
+      const container = document.getElementById("pageContent") as HTMLElement;
+      render(
+        html`<h1
+          style="position: absolute; max-width: 100%; top: 50%; transform: translate(0, -50%);"
+          data-role="notify-auth-success"
+        >
+          You may close this page.
```

Comment:
i18n?

Reply:
We have other screens like this and I've just copied the code for now. I'll extract this in a component and extract the i18n text separately if OK for you.

Reply:
Sure